tests/installation/module_registration/register_modules.pm

```diff
@@ -0,0 +1,33 @@
+# SUSE's openQA tests
+#
+# Copyright 2021 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Register Application module
+#          in "Extension and Module Selection" dialog
+# Medium:  Online (you see the "Hide Development versions" checkbox)
+#
+# Maintainer: QA SLE YaST team <qa-sle-yast@suse.de>
+
+use base 'y2_installbase';
+use strict;
+use warnings;
+use testapi;
+
+sub run {
+    $testapi::distri->get_module_registration()->register_modules([qw(workstation basesystem development legacy containers python3 desktop)]);
+    save_screenshot;
+    my $timeout = 2400 * get_var('TIMEOUT_SCALE', 1);
+    $testapi::distri->get_module_regcode()->wait_regcode_page({
```

Comment:
this logic should be hidden inside the controller, is not part of the test to know that here we need to wait longer, although we should pass the timeout.
I would expect something like `add_separate_regcode($code, $timeout)`.